lib/spack/spack/main.py

```diff
@@ -444,8 +444,9 @@ def make_argument_parser(**kwargs):
         "--config-scope",
         dest="config_scopes",
         action="append",
-        metavar="DIR",
-        help="add a custom configuration scope",
+        metavar="DIR|ENV",
+        help="add directory or environment as read-only configuration scope, without activating "
```

Comment:
This should be shorter, ideally so that `spack -H` doesn't exceed 80 chars (but under 90 would be ok).  We should really enforce this for argparse descriptions so that they don't wrap horribly.